### PR TITLE
PT-512 Display history of parameters in device settings

### PR DESCRIPTION
**Pull request by CedricPerotto**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
   "scripts": {
     "apidocs": "./node_modules/.bin/jsdoc2md",
     "browser-tests": "NODE_ENV=test ./node_modules/.bin/karma start --browsers Chrome",
-    "build": "NODE_ENV=production npm run clean && ./node_modules/.bin/webpack --config package.config.js",
+    "build": "NODE_ENV=production npm run clean && NODE_ENV=production ./node_modules/.bin/webpack --config package.config.js",
```
Comment (ydrisR) [inline package.json:12]: Do you need to reset NODE_ENV twice? Or is it a typo?

**Review by ydrisR — APPROVED** [review]
LGTM, I tested the feature, inside blip and it works as expected.
Just a note on package.json (typo?)

File: package.json (hunk 1 above)
Comment (CedricPerotto) [reply package.json:12]: > Do you need to reset NODE_ENV twice? Or is it a typo?

We need it for the travis build

**Comment by ydrisR** [comment]
@yvon-dblg merging this PR will fix the build issue we have with PR #15

**Pull request merged**